src/pages/recipe/Recipe.js

```diff
@@ -1,29 +1,31 @@
-import { useParams } from 'react-router-dom'
-import { useFetch } from '../../hooks/index'
+import { useParams } from "react-router-dom";
+import { useFetch, useTheme } from "../../hooks/index";
 
 // styles
-import './Recipe.css';
+import "./Recipe.css";
 
 export default function Recipe() {
+	const { id } = useParams();
+	const url = `http://localhost:3000/recipes/${id}`;
```

Comment:
Move this url to a separate file

Reply:
url is for testing purposes only, will move all data to firebase once next.